app/api/routers/breeze_buddy/templates/__init__.py

```
@@ -240,3 +241,10 @@ async def delete_template_by_id(
     require_admin(current_user)
 
     return await delete_template_handler(template_id, current_user)
+
+
+@router.get("/templates/configurations/options")
+async def get_configuration_options(
+    current_user: UserInfo = Depends(get_current_user_with_rbac),
```

Ruff (0.15.7)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)